src/tagstudio/core/global_settings.py

```diff
@@ -44,6 +44,7 @@ class Theme(Enum):
 class GlobalSettings(BaseModel):
     language: str = Field(default="en")
     open_last_loaded_on_startup: bool = Field(default=True)
+    auto_generate_sound_thumbnail: bool = Field(default=False)
```

Comment:
Suggested name change for the setting based on the main review feedback
```suggestion
    generate_thumbs: bool = Field(default=False)
```